msyncd/synchronizer.h

```diff
@@ -446,6 +455,7 @@ private slots:
 #endif
 
     QDBusInterface *iSyncUIInterface;
+    _GSettings *iSettings;
```

Comment:
I can not include "glib.h" here because of conflicts with qt keywords 'signal/slot/etc..', to avoid include id I did a forward declaration of the struct, I can not do it with GSettings.

I can not use qt flag 'no_keywords' here because all code rely on these keywords.